parser/build.rs

```diff
@@ -3,6 +3,10 @@ extern crate lalrpop;
 fn main() {
     ::std::env::set_var("LALRPOP_LANE_TABLE", "disabled");
 
-    lalrpop::Configuration::new().process_current_dir().unwrap();
-    println!("cargo:rerun-if-changed=src/grammar.lalrpop");
+    let out_dir = ::std::env::var("OUT_DIR").unwrap();
+    lalrpop::Configuration::new()
+        .use_cargo_dir_conventions()
+        .process_file("src/grammar.lalrpop")
+        .unwrap();
+    println!("cargo:rerun-if-changed={}/grammar.rs", out_dir);
```

Comment:
Why not:

```rust
println!("cargo:rerun-if-changed=src/grammar.lalrpop");
```
?

Reply:
That is what it was before... Must have copy pasted and accidentally changed it. (The way it is now won't even work, as the .rs ONLY changes if the build script is run 🤦‍♂️

Reply:
That's what I was thinking... 😛